.github/workflows/ci_tests.yml

```diff
@@ -9,16 +9,29 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - name: Update submodules
-      run: |
-        git submodule init
-        git submodule update
-    - name: Get Cmake version
+    - name: apt-get update
+      run |
+        sudo apt-get update --fix-missing
+
+    - name: Workaround - install CMake 3.25.2 since 3.26.0 doesn't work
+      run |
+        sudo apt remove cmake
+        sudo apt purge --auto-remove cmake
+        wget http://www.cmake.org/files/v3.25/cmake-3.25.2.tar.gz
+        tar xf cmake-3.25.2.tar.gz
+        cd cmake-3.25.2
+        ./configure
+        make
+        sudo make install
+        hash -r
```

Comment:
Just out of curiosity, why are you running this? Is it necessary for cmake? I suppose it is?

Reply:
Yes, we have to build cmake from source - at least I didn't found historical .dep packages for cmake.